RECRUITMENT.md

```diff
@@ -14,11 +14,21 @@ In return, for permanent employees, we can offer you:
 - 🛫 All expenses paid travel to work events and conferences
 - 💖 Fantastic, helpful and friendly company culture
 
+🌍 Somewhere else in the world? We still want to hear from you! We always endeavour to look after our team, no matter where you reside.
+
 ### 💻 The technical
 
 Show us the code! We've put together a technical test below to help you show the best of who you are to us. This is what you need to do.
 
-We'd like you to build a basic React Native app with one screen and an input for an Ethereum address and a numberical input.
+We'd like you to build a basic React Native app that utilises the Etherspot platform with one screen minimum (you can add more if you think it works for your app), an input for an Ethereum address and a numberical input for the send amount.
+
+**About the Etherspot Platform**
+
+The Etherspot Platform allows developers and organisations to quickly bootstrap their apps and dApps, with contract management and blockchain interaction handled by the platform itself. All interaction of the Etherspot Platform is handled by the Etherspot SDK. Here are some links that you may find useful:
+
+- [Etherspot Documentation](https://docs.etherspot.dev)
+- [Etherspot SDK Reference](https://sdk.etherspot.dev)
```

Comment:
wdyt
```suggestion
- [Etherspot SDK Playground](https://sdk.etherspot.dev)
```

Reply:
That's _actually_ the SDK reference. The SDK Playground is at https://try.etherspot.dev, but on that note i'll add that as a separate link

Reply:
Updated 👍